src/gui/shell.cpp

```diff
@@ -882,9 +884,10 @@ void Shell::handleExtGuiOption(const QString& name, const QVariant& value)
 void Shell::handleSetOption(const QString& name, const QVariant& value)
 {
 	if (name == "guifont") {
-		setGuiFont(value.toString(), false /*force*/, false /*setOption*/);
+		setGuiFont(value.toString(), false /*force*/, false /*setOption*/, false);
 	} else if (name == "guifontset") {
 	} else if (name == "guifontwide") {
+		setGuiFont(value.toString(), false /*force*/, false /*setOption*/, true);
```

Comment:
I think the logic here needs some modification. The `guifontwide` option is supposed to be a list of fonts.

From `:help guifontwide`:
> When not empty, specifies a comma-separated list of fonts to be used for double-width characters.  The first font that can be loaded is used.